gui/backend/service/cluster/cluster.go

```diff
@@ -0,0 +1,58 @@
+package cluster
+
+import (
+	"errors"
+	"net/http"
+	"strings"
+
+	"github.com/cubefs/cubefs/proto"
+	"github.com/gin-gonic/gin"
+
+	"github.com/cubefs/cubefs/console/backend/helper/httputils"
+)
+
+func Get(c *gin.Context, addr string) (*proto.ClusterView, error) {
+	reqUrl := "http://" + addr + proto.AdminGetCluster
+	req, err := httputils.DoRequestOvertime(c, reqUrl, http.MethodGet, nil, nil)
+	if err != nil {
+		return nil, err
+	}
+	output := httputils.Output{Data: &proto.ClusterView{}}
+	_, err = httputils.HandleResponse(c, req, err, &output)
+	if err != nil {
+		return nil, err
+	}
+	if output.Code != proto.ErrCodeSuccess {
+		return nil, errors.New(output.Msg)
+	}
+	return output.Data.(*proto.ClusterView), nil
+}
```

Comment:
May I ask why you don't choose to use sdk here, but rewrite the http request? In other words, have you considered using the sdk?

Reply:
good idea. we consider use the sdk laster.